spec/models/claim/base_claim_spec.rb

```diff
@@ -112,6 +112,42 @@ class MockSteppableClaim < Claim::BaseClaim
     end
   end
 
+  describe '#clar?' do
+    subject { claim.clar? }
+
+    context 'when the claim is AGFS scheme 12' do
+      let(:claim) { create :advocate_claim, :agfs_scheme_12 }
+
+      it { is_expected.to be_truthy }
+    end
+
+    context 'when the claim is AGFS scheme 11' do
+      let(:claim) { create :advocate_claim, :agfs_scheme_11 }
+
+      it { is_expected.to be_falsey }
+    end
+
+    context 'when the claim is LGFS CLAR' do
+      let(:claim) { create :litigator_claim, :clar }
+
+      it { is_expected.to be_truthy }
+    end
+
+    context 'when the claim is LGFS pre-CLAR' do
+      let(:claim) { create :litigator_claim, :pre_clar }
+
+      it { is_expected.to be_falsey }
+    end
+
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/4011/files#r652792036)